GVRf/Framework/framework/src/main/java/org/gearvrf/GVRCamera.java

```diff
@@ -94,6 +94,7 @@ public void setBackgroundColor(int color) {
      * Meaningful parameter values are from 0 to 1, inclusive: values
      * {@literal < 0} are clamped to 0; values {@literal > 1} are clamped to 1.
      */
+    @Deprecated
```

Comment:
Why is this getting deprecated?

If this one is deprecated there is also the `public void setBackgroundColor(int color)` function as well.